twind.config.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
 import { Options } from "$fresh/plugins/twindv1.ts";
-import { defineConfig, Preset } from "twind";
+import { defineConfig, Preset } from "https://esm.sh/@twind/core@1.1.3";
```

Comment:
Can you please revert? I don't think this is needed.

Reply:
Can you please confirm this here?
I needed to change this because once I upgraded fresh to 1.2, defineConfig and Preset were not defined in "twind" anymore.
This guide here (https://fresh.deno.dev/docs/examples/using-twind-v1) also imports defineConfig from core@1.1.3